examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -53,6 +53,16 @@
 
 /**
  * End-to-end tests of WordCount.
+ *
+ * <p>Input text document is available from the following sources:
```

Comment:
The above problem turns out that text file wasn't in right resources directory. 

However, running WordCount through e2e test on GCP(using TestDataflowRunner) or locally(using Spark/Flink runner) requires dynamically resolve file path according to different runner/fs in runtime. Otherwise, test with default input path can fail one runner or the others. I'd like to use different file path as default input according to runners.